test/liquidate/liquidate.ts

```diff
@@ -45,6 +58,7 @@ describe('Liquidate Validator Tests', () => {
     await trackGas(ssvNetworkContract.connect(helpers.DB.owners[4]).registerValidator(helpers.DataGenerator.publicKey(9), clusterResult1.clusterId, helpers.DataGenerator.shares(0)), [GasGroup.REGISTER_VALIDATOR_EXISTING_POD]);
   });
 
+  // Test to be removed - duplicate
   it('Liquidate and register validator in disabled pod', async () => {
```

Comment:
Why to remove where is the repeat?

Reply:
This test "Liquidate and register in disabled pod" is the same as the test above "Liquidate and register validator in disabled pod"